operations/mimir-mixin/alerts/alerts-utils.libsonnet

```
@@ -8,4 +8,11 @@
   // The alert name is prefixed with the product name (eg. AlertName -> MimirAlertName).
   alertName(name)::
     $._config.product + name,
+
+  // The mixin allow specialism of the job selector depending on if its a single binary
+  // deployment or a namespaced one.
+  jobMatcher(job)::
+    if $._config.singleBinary
+    then 'job=~"$job"'
+    else '%s=~"$cluster", job=~"($namespace)/(%s)"' % [$._config.per_cluster_label, job],
```

This has been copy-pasted from the dashboard util, but in alerts we don't have the dashboard variables like `$job`, `$cluster` or `$namespace`. Here we want to match any namespace, so it's easier:


```suggestion
    'job=~".*/%s"' % job,
```